kroxylicious-operator/DEV_GUIDE.md

```diff
@@ -90,4 +90,19 @@ Note that on its own the proxy won't try to connect to the Kafka cluster.
 
 # Testing
 
-To test things properly you'll need to point your virtual clusters at a running Kafka and also run a Kafka client so the proxy is handling some load.
\ No newline at end of file
+To test things properly you'll need to point your virtual clusters at a running Kafka and also run a Kafka client so the proxy is handling some load.
+
+## System Tests
+
+THe Kroxylicious system test suite has been updated to use the operator to deploy resources, so tests can be written in
+java and executed locally in one's IDE. They do however require access to a Kubernetes clusters (usually minikube) and
+helm, with the appropriate RBAC permissions to install operators and provision resources.
```

Comment:
Not for this PR, but we should call out exactly what the developer needs to bring/do to run the system tests.